app/views/components/multi-select/index.html

```diff
@@ -15,10 +15,10 @@ <h1 class="govuk-heading-xl">
         <span class="govuk-caption-xl">Components</span> Multi-select
       </h1>
 
-      <table class="govuk-table">
+      <table class="govuk-table" data-module="moj-multi-select" data-multi-select-checkbox="#select-all">
         <thead class="govuk-table__head">
           <tr class="govuk-table__row">
-            <th class="govuk-table__header moj-multi-select__select-all-container" scope="col"></th>
+            <th class="govuk-table__header" scope="col" id="select-all"></th>
```

Comment:
Will this cause HTML Validation issues if there are multiple tables with the same ID? Would it be better as a default first column or override with optional defined property?

Reply:
The selector is kind of irrelevant because it's just whatever's targetted by `data-multi-select-checkbox` on line 18. I agree that ID is a bad example though, and can change it to a data-* attribute.

I didn't want to make any assumptions about where the column would be to make sure we don't break any production sites. Insisting you define and specify one felt like the most clear and flexible rule.

Reply:
Yeah that makes sense, nice one, all looking good to me